sdk/Sdk.Generators/ExtensionStartupRunnerGenerator.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+namespace Microsoft.Azure.Functions.Worker.Sdk.Generators
+{
+    /// <summary>
+    /// Generates a class with a method which has code to call the "Configure" method
+    /// of each of the participating extension's "IWorkerExtensionStartup" implementations.
+    /// Also adds the assembly attribute "WorkerExtensionStartupCodeExecutorInfo"
+    /// and pass the information(the type) about the class we generated.
+    /// We are implementing the IWorkerExtensionStartup interface on the generated class.
+    /// (This is the same interface exetension authors will implement for their extension specific startup code)
+    /// We need the same signature as the extension's implementation as our class is an uber class which internally
+    /// calls each of the extension's implementations.
+    /// </summary>
+
+    // Sample code generated (with one extensions particiapating in startup hook)
+    // There will be one try-catch block for each extension particiapating in startup hook.
```

Comment:
fix spelling of "participating" (2x here)

Reply:
Fixed type.